src/pages/summer/Login/LoginInput/LoginInput.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+import './LoginInput.scss';
+
+class LoginInput extends React.Component {
+  state = {
+    idInput: '',
+    pwInput: '',
+    idVld: false,
+    pwVld: false,
+  };
+
+  gotoMain = e => {
+    const { IP_ADDRESS } = this.props;
+    e.preventDefault();
+
+    fetch(`http://${IP_ADDRESS}:8000/users/signin`, {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.idInput,
+        password: this.state.pwInput,
+        // name: '포켓몬 지우',
+        // phone_number: '010-0000-0000',
+      }),
```

Comment:
```suggestion
      body: JSON.stringify({ email, password }),
```
객체의 키와 값의 이름이 같을 경우 위와 같이 생략해줄 수 있습니다